### PR TITLE
Fix Libs config to match MILO 

### PR DESCRIPTION
**Pull request by Blainegunn**
Fix Libs config to match MILO

### CODE REVIEW

**Comment by TsayAdobe** [comment]
Please review the test cases in `scripts.jest.js` and `utils.jest.js`.

File: test/scripts/scripts.jest.js [hunk 1]
```diff
@@ -118,7 +118,7 @@ describe('Test scripts', () => {
       await require('../../acrobat/scripts/scripts.js');
       await delay(100);
       const config = await getConfig();
-      expect(config.miloLibs).toEqual('http://localhost:6456/libs');
+      expect(config.miloLibs).toEqual('/libs');
```
Comment (TsayAdobe) [inline test/scripts/scripts.jest.js:121]: When `milolibs=local`, it should be  `http://localhost:6456/libs`

File: test/scripts/utils.jest.js [hunk 2]
```diff
@@ -34,7 +34,7 @@ describe('Test utils.js', () => {
       'https://www.stage.adobe.com?milolibs=local'
     );    
     const libs = setLibs('/libs');
-    expect(libs).toBe('http://localhost:6456/libs');
+    expect(libs).toBe('/libs');
```
Comment (TsayAdobe) [inline test/scripts/utils.jest.js:37]: When `milolibs=local`, it should be `http://localhost:6456/libs`

**Review by TsayAdobe — CHANGES_REQUESTED** [review]
We need to enable devs using a local milo.

**Review by joaquinrivero — APPROVED** [review]

**Pull request closed without merging**